kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/base/BaseOperation.java

```diff
@@ -350,11 +350,22 @@ public T apply(T resource) {
   }
 
   @Override
-  public T createOrReplace(T item) {
+  public T createOrReplace(T... items) {
+    T item = getItem();
```

Comment:
Yeah that is exactly why. This enables `client.pods().load(...).createOrReplace()` whereas before you'd have to pass something in to `createOrReplace`.